src/grid/_grid.scss

```diff
@@ -87,10 +87,7 @@
 
   // Define partial sizes for columnNumber < totalColumns.
   @for $i from 1 through ($grid-phone-columns - 1) {
-    .mdl-cell--#{$i}-col {
-      @include partial-size($i, $grid-phone-columns, $grid-phone-gutter);
-    }
-
+    .mdl-cell--#{$i}-col,
     .mdl-cell--#{$i}-col-phone.mdl-cell--#{$i}-col-phone {
```

Comment:
Just noticed multiple selector declaration. Any reason behind this? Or I can change this into ```.mdl-cell-#{i}-col-phone``` only.